packages/esm-patient-chart-app/src/banner/contact-details/contact-details.scss

```diff
@@ -7,11 +7,18 @@
   width: 100%;
   border-top: 1px solid $ui-03;
 }
+.link{
+  text-decoration-line: none;
+  text-transform: capitalize;
+}
 
 .heading {
   @include type.type-style("heading-compact-01");
   color: $text-02;
 }
+.heading button {
+  float: right;
```

Comment:
Please don't use float

Either use flex with `justify-content: space-between` in the parent element or give `margin-left: auto` to the button, whichever works.

Thanks!